components/notification/demo/with-btn.md

```diff
@@ -1,31 +1,39 @@
 ---
 order: 4
-title: 自定义
+title: 
+  zh-CN: 自定义
+  en-US: Custom
 ---
 
+## zh-CN
+
 自定义关闭按钮的样式和文字。
 
+## en-US
+
+To customize the style or font of the close button.
+
 ````jsx
 import { Button, notification } from 'antd';
 
 const close = function () {
-  console.log('我被默认的关闭按钮关闭了！');
+  console.log('I am closed by the default close button.');
 };
 
 const openNotification = function () {
   const key = `open${Date.now()}`;
   const btnClick = function () {
-    // 隐藏提醒框
+    // to hide notification box
     notification.close(key);
   };
   const btn = (
     <Button type="primary" size="small" onClick={btnClick}>
-      自定义关闭按钮并触发回调函数
+      To speicify a function that will be called after clicking the close button
     </Button>
   );
   notification.open({
-    message: '这是标题',
-    description: '这是提示框的文案这是提示框示框的文案这是提示是提示框的文案这是提示框的文案',
+    message: 'This is the title',
+    description: 'This is the content of the notification.This is the content of the notification.This is the content of the notification.',
```

Comment:
description: 'A function will be called after the notification is closed (automatically, after the 'duration' time of manually).',